app/src/main/java/a3/com/convo/activities/PlayGameActivity.java

```diff
@@ -16,10 +19,10 @@
     private FriendsFragment friendsFrag;
     private ModeFragment modeFrag;
     private GameFragment gameFrag;
+    private ConclusionFragment conclusionFrag;
     private Fragment playGameFrag;
```

Comment:
We should NOT be saving fragments in memory even if it is an activity. This leads to memory leaks in android. Lets take this up as a separate issue. But lets fix this as soon as possible.

Reply:
OK thanks, I think Andrea fixed this in her commits